version/news/NEWS-2023.03.0-cherry-blossom.md

```diff
@@ -73,3 +77,5 @@
 - Fixed the frequent `No such file or directory` errors logged by `rserver-monitor` when on a system with cgroups v2 (#10914, rstudio/rstudio-pro#2683)
 - Support Kubernetes v1.24 (rstudio/rstudio-pro#4064)
 - Security fix: User ID is now set by the server when running Workbench Jobs created by VS Code (rstudio/rstudio-pro#4255)
+- Fixed issue where sessions and jobs running on Slurm could lose environment variables in a load-balanced cluster or after a restart (rstudio/launcher#533)
+- Fixed issue where available Slurm partitions would not be listed under certain user profile configurations (rstudio/launcher#520)
```

Comment:
```suggestion
- Fixed connection issues with SSL-enabled sessions running on Slurm in a load-balanced cluster or after a Launcher restart (rstudio/rstudio-pro#4098)
```

* I'd suggest pointing to the Workbench-specific issue here, and couching the changes as they apply to Workbench.

* https://github.com/rstudio/launcher/issues/520 never affected customers because it was fixed in the same release that the issues started appearing in (see https://github.com/rstudio/rstudio-pro/issues/3906).

Reply:
Did you mean to also delete the `Slurm partition` news entry in this suggestion?